Fixing typo

aliuy opened the pull request:
There are [10,000 ticks per millisecond (instead of 1,000)](https://msdn.microsoft.com/en-us/library/system.timespan.tickspersecond(v=vs.110).aspx)

azurecla commented:
Hi __@harelbr__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

aliuy committed:
Fixing typo

There are [10,000 ticks per millisecond (instead of 1,000)](https://msdn.microsoft.com/en-us/library/system.timespan.tickspersecond(v=vs.110).aspx)
